### DISCUSSION TITLE
Invalid name Error on macOS

### DISCUSSION THREAD

**Discussion by ASQABenny** [webdriverio/webdriverio#9089] [Q&A]
Hey guys

My test automation with wdio for a website works fine on Windows Chrome.
Now I tried to bring it to macOS Safari but when I run it, it shows me the error message: npm ERR: Invalid name: "Project Name".

Please help :)

**Comment by christian-bromann** [comment]
> npm ERR: Invalid name: "Project Name".

Please provide a whole stack trace, this doesn't help much.
